soh/src/overlays/actors/ovl_Boss_Ganon/z_boss_ganon.c

```diff
@@ -1209,19 +1209,15 @@ void BossGanon_ShatterWindows(u8 windowShatterState) {
 }
 
 void BossGanon_DeathAndTowerCutscene(BossGanon* this, GlobalContext* globalCtx) {
-    const bool originalBlood = CVar_GetS32("gOriginalBlood", 1);
+    static Color_RGBA8 bloodPrimColor = { 0, 120, 0, 255 };
+    static Color_RGBA8 bloodEnvColor = { 0, 120, 0, 255 };
 
-    static Color_RGBA8 bloodPrimColor = { 120, 0, 0, 255 };
-    static Color_RGBA8 bloodEnvColor = { 120, 0, 0, 255 };
+    if(CVar_GetS32("gRedGanonBlood", 0) == 1) {
```

Comment:
Instead of doing `== 1` I would do `!= 0` or just `CVar_GetS32("gRedGanonBlood", 0)`. A true value in a boolean can be thought of as anything that is non zero.

Reply:
I picked up the whole evaluating bools thing as it seemed to already be a bit of a convention, but I agree that just using them directly is much neater, so with your approval I went with that.